web/src/features/fireWeather/components/tables/StationComparisonTable.tsx

```diff
@@ -0,0 +1,482 @@
+    title.push(
+      <div>
+        prediction: {value.datetime}, precipitation:{' '}
+        {value.delta_precipitation?.toFixed(PRECIP_VALUES_DECIMAL)} mm (model:{' '}
+        {value.model_run_datetime})
+      </div>
+    )
+  })
+  return (
+    <ToolTip title={title} aria-label="precipitation">
+      <div className={precipitationClassName}>
+        {precipitation &&
+          typeof precipitation.precipitation === 'number' &&
+          `${precipitation.precipitation.toFixed(PRECIP_VALUES_DECIMAL)} mm`}
+      </div>
+    </ToolTip>
+  )
+}
+
+const formatDewPoint = (dewpoint: number | null | undefined) => {
+  return (
+    <div>
+      {typeof dewpoint === 'number' &&
+        `${dewpoint.toFixed(DEW_POINT_VALUES_DECIMAL)}${String.fromCharCode(176)}C`}
+    </div>
+  )
+}
+
+const SubHeadings = (index: number, className: string) => [
+  <TableCell key={`observered-${index}`} className={className}>
+    Observed
+  </TableCell>,
+  <TableCell key={`forecast-${index}`} className={className}>
+    Forecast
+  </TableCell>,
+  <TableCell key={`HRDPS-${index}`} className={className}>
+    HRDPS
+  </TableCell>,
+  <TableCell key={`RDPS-${index}`} className={className}>
+    RDPS
+  </TableCell>,
+  <TableCell key={`GDPS-${index}`} className={className}>
+    GDPS
+  </TableCell>
+]
+
+const StationComparisonTable = (props: Props) => {
+  const classes = useStyles()
+  // format the date to match the ISO format in the API for easy comparison.
+  const noonDate = formatDateInUTC0(props.timeOfInterest)
+  return (
+    <Paper className={classes.paper}>
+      <Typography component="div" variant="subtitle2">
+        Station comparison for {formatDateInPST(noonDate)} PDT
+      </Typography>
+      <Paper>
+        <TableContainer>
+          <Table
+            stickyHeader
+            size="small"
+            aria-label="sortable wx table"
+            data-testid="station-comparison-table"
+          >
+            <TableHead>
+              <TableRow>
+                <TableCell></TableCell>
+                <TableCell className={classes.darkColumnHeader} colSpan={5}>
+                  Temperature
+                </TableCell>
+                <TableCell className={classes.lightColumnHeader} colSpan={5}>
+                  Relative Humidity
+                </TableCell>
+                <TableCell className={classes.darkColumnHeader} colSpan={5}>
+                  Wind Speed + Direction
+                </TableCell>
+                <TableCell className={classes.lightColumnHeader} colSpan={5}>
+                  Precipitation
+                </TableCell>
+                <TableCell className={classes.darkColumnHeader}>Dew point</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Weather Stations</TableCell>
+                {['temp', 'rh', 'wind', 'precip'].map((value, index) => {
+                  return SubHeadings(index, classes.darkColumnHeader)
```

Comment:
very low-priority and pedantic, but the subheadings always have dark backgrounds, whereas I would've expected the subheading background to match its corresponding heading's background

Reply:
Ambrose where are you!